src/pudl/extract/ferc1.py

```diff
@@ -266,7 +266,7 @@ def get_file(self, year: int, filename: str):
         """Opens given ferc1 file from the corresponding archive."""
         if year not in self._cache:
             self._cache[year] = self.datastore.get_zipfile_resource(
-                "ferc1", year=year, data_format="DBF"
+                "ferc1", year=year, data_format="dbf"
```

Comment:
@zschira let's standardize on one or the other case in the archives here.

Reply:
i added [an issue](https://github.com/catalyst-cooperative/pudl-archiver/issues/12) in maybe the right repo